option.go

```
@@ -507,3 +531,10 @@ func (option *Option) shortAndLongName() string {
 
 	return ret.String()
 }
+
+func (option *Option) isValidValue(arg string) bool {
```

I think this should probably return an error